src/main/java/com/mysema/codegen/model/TypeCategory.java

```diff
@@ -50,7 +50,7 @@
     /**
      *
      */
-    COMPARABLE(SIMPLE),
+    COMPARABLE(SIMPLE, java.util.UUID.class.getName()),
```

Comment:
This would bleed the change to the `COMPARABLE` category into `DATE`, `DATETIME`...
So this is not the correct solution.